### PR TITLE
Css style signup screen

### PR DESCRIPTION
**Pull request by hancel-avila**
# Description
What did you implemented/Why did you implemented this:
 - We declared the colors we are going to use in the User Interface. Why? This way we define the style on future layouts. Also, brighter colors will be used to bring the attention on relevant components.
 - We made the design for the SignUp screen applying CSS. Why? We created a friendly navigation interface for the user, this way they can fill up all the info required without being lost.
 
image

### CODE REVIEW

**Review by carloseag1609 — APPROVED** [review]
Looks fine

**Review by cortezroberto — APPROVED** [review]
Good!

**Review by DemisRincon — COMMENTED** [review]
Please move your provider and store to root/App.js it should be a global state, giving the provider a unique component will not perform the expected functionality please read the redux documentation or search advice from a mentor.

Please try to wait at least for a Mentor Review, if you don't receive a review soon please contact to @DemisRincon  or Jose Tamara both are active in the group all-day

**Comment by carloseag1609** [comment]
> Please move your provider and store to root/App.js it should be a global state, giving the provider a unique component will not perform the expected functionality please read the redux documentation or search advice from a mentor.
> 
> Please try to wait at least for a Mentor Review, if you don't receive a review soon please contact to @DemisRincon or Jose Tamara both are active in the group all-day

Thanks for the comment Demis, we forgot that. We'll move the Provider component and the store in the next task (navigation)

File: FlightReservation/src/screens/SignUp.js [hunk 1]
```diff
@@ -38,45 +48,96 @@ const SignUp = function () {
...
+                </View>
+                <View>
+                  {/* <Button
+                    rounded
+                    disabled={!formProps.isValid || loading}
+                    onPress={formProps.handleSubmit}
+                    title="Sign up"
+                  />
+                  <Text style={style.inferior_content_text}>or</Text>
+                  <Button title="Sign up with Google" /> */}
```
Comment (YilKanda) [inline FlightReservation/src/screens/SignUp.js:93]: If they don't occupy it, delete it.

File: FlightReservation/src/screens/SignUp.js [hunk 2]
```diff
@@ -38,45 +48,96 @@ const SignUp = function () {
...
+                  <View
+                    style={{
+                      flex: 1,
+                      justifyContent: 'flex-end',
+                      paddingBottom: 40,
+                    }}
```
Comment (YilKanda) [inline FlightReservation/src/screens/SignUp.js:99]: It is recommended that all styles are separated in a style file.

File: FlightReservation/src/screens/SignUp.js [hunk 3]
```diff
@@ -38,45 +48,96 @@ const SignUp = function () {
...
+                      <Text style={{color: colors.white, fontSize: 18}}>
+                        Sign Up
+                      </Text>
+                    </View>
+                  </Pressable>
+
+                  <Text
+                    style={[
+                      style.lower_content_text,
+                      {paddingTop: 10, paddingBottom: 10},
```
Comment (YilKanda) [inline FlightReservation/src/screens/SignUp.js:114]: It is recommended that all styles are separated in a style file.

File: FlightReservation/src/screens/SignUp.js [hunk 4]
```diff
@@ -38,45 +48,96 @@ const SignUp = function () {
...
+                    ]}>
+                    or
+                  </Text>
+
+                  <Pressable
+                    disabled={!formProps.isValid || loading}
+                    onPress={formProps.handleSubmit}>
+                    <View style={style.btn}>
+                      <Text style={{color: colors.white, fontSize: 18}}>
```
Comment (YilKanda) [inline FlightReservation/src/screens/SignUp.js:123]: It is recommended that all styles are separated in a style file.

File: FlightReservation/src/screens/SignUp.js [hunk 5]
```diff
@@ -38,45 +48,96 @@ const SignUp = function () {
...
+                        Sign Up with Google
+                      </Text>
+                    </View>
+                  </Pressable>
+
+                  <View style={style.lower_content_text}>
+                    <Text style={{paddingTop: 20}}>
```
Comment (YilKanda) [inline FlightReservation/src/screens/SignUp.js:130]: It is recommended that all styles are separated in a style file.

File: FlightReservation/src/screens/SignUp.js [hunk 6]
```diff
@@ -38,45 +48,96 @@ const SignUp = function () {
...
+                      flex: 1,
+                      justifyContent: 'flex-end',
+                      paddingBottom: 40,
+                    }}
+                  />
+                  <Pressable
+                    disabled={!formProps.isValid || loading}
+                    onPress={formProps.handleSubmit}>
+                    <View style={style.btn}>
+                      <Text style={{color: colors.white, fontSize: 18}}>
```
Comment (YilKanda) [inline FlightReservation/src/screens/SignUp.js:105]: It is recommended that all styles are separated in a style file.

**Review by YilKanda — COMMENTED** [review]
Try to use internalization instead of static texts.

You can make use of libraries like this:

[react-native-i18n](https://www.npmjs.com/package/react-native-i18n)

**Pull request closed without merging**